packages/devel/configtools/package.mk

```diff
@@ -1,17 +1,5 @@
-################################################################################
-#  This program is free software: you can redistribute it and/or modify
-#  it under the terms of the GNU General Public License as published by
-#  the Free Software Foundation, either version 2 of the License, or
-#  (at your option) any later version.
-#
-#  This program is distributed in the hope that it will be useful,
-#  but WITHOUT ANY WARRANTY; without even the implied warranty of
-#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#  GNU General Public License for more details.
-#
-#  You should have received a copy of the GNU General Public License
-#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
-################################################################################
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
```

Comment:
@stefansaraev as the package came from your repo and you did not include any copyright `# Copyright (C) 2016 seo` are you okay with that or should we add you here ?

Reply:
I dont care about copyrights, keep it as-is :)

Reply:
Thanks @stefansaraev